features/users/user_delete.feature

```diff
@@ -73,9 +73,8 @@ Scenario: Delete a user with a collection
     And a user account should not exist for "moderator"
   When I go to the collections page
   Then I should see "fake"
-    # TODO: And a caching bug is fixed...
-    # And I should see "orphan_account"
-    # And I should not see "moderator"
+    And I should see "orphan_account"
+    And I should not see "moderator"
```

Comment:
This already passes even without the changes in this PR. I could not find out what fixed it, the original comment is from 2013.